src/menu.js

```diff
@@ -0,0 +1,116 @@
+
+
+  const divTwo = document.createElement('div');
+  const img1 = document.createElement('img');
+  img1.setAttribute('class', 'chapo');
+  img1.src = 'https://img-global.cpcdn.com/recipes/30813288c94f72bf/1200x630cq70/photo.jpg';
+
+  const menuParagraphTwo = document.createElement('p');
+  menuParagraphTwo.setAttribute('id', 'text');
+  menuParagraphTwo.innerHTML = 'Chapati & Maharagwe';
+
+  const divThree = document.createElement('div');
+  const img2 = document.createElement('img');
+  img2.setAttribute('class', 'chapo');
+  img2.src = 'https://img-global.cpcdn.com/recipes/30813288c94f72bf/1200x630cq70/photo.jpg';
+
+  const menuParagraphThree = document.createElement('p');
+  menuParagraphThree.setAttribute('id', 'text');
+  menuParagraphThree.innerHTML = 'Chapati & Maharagwe';
+
+
+  const menuItems1 = document.createElement('div');
+  menuItems1.setAttribute('class', 'menu-items');
+
+
+  const divFour = document.createElement('div');
+  const img4 = document.createElement('img');
+  img4.setAttribute('class', 'chapo');
+  img4.src = 'https://img-global.cpcdn.com/recipes/30813288c94f72bf/1200x630cq70/photo.jpg';
+
+  const menuParagraphFour = document.createElement('p');
+  menuParagraphFour.setAttribute('id', 'text');
+  menuParagraphFour.innerHTML = 'Chapati & Maharagwe';
+
+
+  const divFive = document.createElement('div');
+  const img5 = document.createElement('img');
+  img5.setAttribute('class', 'chapo');
+  img5.src = 'https://img-global.cpcdn.com/recipes/30813288c94f72bf/1200x630cq70/photo.jpg';
+
+  const menuParagraphFive = document.createElement('p');
+  menuParagraphFive.setAttribute('id', 'text');
+  menuParagraphFive.innerHTML = 'Chapati & Maharagwe';
+
+  const divSix = document.createElement('div');
+  const img6 = document.createElement('img');
+  img6.setAttribute('class', 'chapo');
+  img6.src = 'https://img-global.cpcdn.com/recipes/30813288c94f72bf/1200x630cq70/photo.jpg';
+
+  const menuParagraphSix = document.createElement('p');
+  menuParagraphSix.setAttribute('id', 'text');
+  menuParagraphSix.innerHTML = 'Chapati & Maharagwe';
+
+
+  menuDishes.appendChild(menuItems);
+  menuItems.appendChild(divOne);
+  divOne.appendChild(img);
+  divOne.appendChild(menuParagraph);
+  menuItems.appendChild(divTwo);
+  divTwo.appendChild(img1);
+  divTwo.appendChild(menuParagraphTwo);
+  menuItems.appendChild(divThree);
+  divThree.appendChild(img2);
+  divThree.appendChild(menuParagraphThree);
+  menuDishes.appendChild(menuItems1);
+  menuItems1.appendChild(divFour);
+  divFour.appendChild(img4);
+  divFour.appendChild(menuParagraphFour);
+  menuItems1.appendChild(divFive);
+  divFive.appendChild(img5);
+  divFive.appendChild(menuParagraphFive);
+  menuItems1.appendChild(divSix);
+  divSix.appendChild(img6);
+  divSix.appendChild(menuParagraphSix);
+  // menuItems.appendChild(img1);
+  // menuItems.appendChild(img2);
+  // menuItems.appendChild(img3);
+  // menuItems.appendChild(img4);
+  // menuItems.appendChild(img5);
```

Comment:
It’s better to delete commented code. If you need to review a previous version of your code, use git history. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

Reply:
noted